### PR TITLE
Portfolio 1

### PR DESCRIPTION
**Pull request by michaelasong**
Updated link style
Fixed body margin
Changed branch name

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @michaelasong , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that doesn't have the readme template provided by microverse. Kindly make sure you add readme to your project following this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)

- Kindly make sure you don't create a new pull request. When changes are required, ensure you push to the same opened Pull request 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+        <img src="image/Union.png" alt="" />
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] for semantic purposes kindly put just the  mobile hamburger inside the `nav tag` and the logo outside of it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      </div>
+      <div class="menuSection">
+        <a href="#" target="_blank">LET'S CONNECT</a>
+        <div class="socialMedia">
+          <img src="image/Twitter.png" alt="" />
+          <img src="image/Linkedin.png" alt="" />
+          <img src="image/Meduim.png" alt="" />
+          <img src="image/Github.png" alt="" />
+          <img src="image/angellist.png" alt="" />
+        </div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please put the social icons in an `ul,` and each of them in a `li`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  <body>
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+        <img src="image/Union.png" alt="" />
+      </nav>
+    </header>
+    <section class="backgroundimg">
+      <div class="menuSection">
+        <h1 class="introduction">I’m Mariane Glad to see you!</h1>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kindly follow the figma font weight and size of the headline section

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @michaelasong 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅figma design followed

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Kossi-stack** [comment]
You have linters errors

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @michaelasong,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**